[Settings/#1] xcode project setting

leemhyungyu opened the pull request:
## 작업한 내용
- xcode project 생성
- scheme 설정
- build configuration 설정
- SPM 설정 (Alamofire, Moya)

## 전달 사항
- xcconfig 보내드릴게요~

## 관련 이슈
- Resolved: #1

leemhyungyu committed:
chore: xcode.proj 생성

leemhyungyu committed:
chore: Main.storyboard 삭제

- 관련 info.plist, Build Setting 수정

leemhyungyu committed:
chore: release, dev scheme & configuration 설정

leemhyungyu committed:
feat: scheme에 따른 xcconfig 설정

- dev scheme -> Debug Configuration
- release scheme -> Release Configuration
- 각 scheme, configuration에 따른 확인을 테스트하기 위해 ViewController Label로 구분

leemhyungyu committed:
chore: SPM 설정

- Alamofire, Moya

dayo2n commented on Meme/Meme.xcodeproj/project.xcworkspace/contents.xcworkspacedata:
이거 모지 ??

leemhyungyu replied:
xcworkspace 사용했을 때 project들 레퍼런스 가지고 있는 파일이라는데 일단 우리 workspace안써서 ignore처리하고 새로 빌드해보니깐 정상적으로 빌드되길래 일단 ignore 처리해놨어!

leemhyungyu committed:
chore: Kingfisher 추가

dayo2n commented on Meme/Meme.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:
xcshareddata gitignore에 넣으면 안되던가 ?!

leemhyungyu replied:
gitignore 디렉터리 설정 잘못돼서 제대로 적용안돼서 다시 설정해놨어~!

dayo2n commented on Meme/Meme/SceneDelegate.swift:
```diff
@@ -0,0 +1,53 @@
+//  Created by 임현규 on 7/15/25.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        let window = UIWindow(windowScene: windowScene)
+        window.rootViewController = ViewController()
+        self.window = window
```
우와 나 씬으로 처음 관리해본당ㅋㅋ

dayo2n reviewed (approved):
헉 갑자기 카톡방에서 노느라 어프루브를 안눌렀닼ㅋㅋㅋ

leemhyungyu committed:
chore: .gitignore 재적용

leemhyungyu committed:
chore: App Target build configuration 설정

- Meme-release -> Release build Configuration
- Meme-dev -> Development build Configuration

leemhyungyu committed:
chore: Meme-dev target info.plist 경로 수정

leemhyungyu commented:
@dayo2n 
추가적으로 ignore처리하니깐 edit scheme으로 build configuration 설정했던게 ignore 처리돼서 Project 설정에서 각 App Target 마다 Configuration 설정하는거로 수정했어~~

빌드 정상적으로 되는거 확인되면 말해줘 머지할게!
